geocoding/location-service/src/app/geocoder_suggester.py

```diff
@@ -1,12 +1,16 @@
 import logging
 import sys
+import json
 
 from src.app.geocoder import Geocoder
 
 h = logging.StreamHandler(sys.stdout)
 logger = logging.getLogger(__name__)
 logger.addHandler(h)
 logger.setLevel(logging.INFO)
+adm1ParsedData = json.load(open('data/adm1_parsed_data.json'))
+adm2ParsedData = json.load(open('data/adm2_parsed_data.json'))
+adm3ParsedData = json.load(open('data/adm3_parsed_data.json'))
```

Comment:
Could we get this from an S3 bucket or API endpoint instead?

Reply:
Yes, I think I'd prefer adding a new issue for this, as this will require configuring mock for AWS for unit and integration testing.